lib/components/common/CustomMappingDropdown.tsx

```diff
@@ -1,877 +1,914 @@
-import React, { useState } from 'react';
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them